src/ILVerify/tests/ILTests/SwitchTests.il

```diff
@@ -0,0 +1,260 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+.assembly extern System.Runtime
+{
+}
+
+.assembly SwitchTests
+{
+}
+
+.class public auto ansi beforefieldinit SwitchTestsType
+       extends [System.Runtime]System.Object
+{
+    .method static public hidebysig void Switch.Int32Value_Valid() cil managed
+    {
+        .maxstack 1
+        .locals init (
+            [0] int32
+        )
+
+        IL_0000: ldc.i4.s 10
+        IL_0001: stloc.0
+
+        IL_0003: ldloc.0
+        IL_0004: switch    (IL_0041, IL_002B, IL_002B, IL_002B, IL_002B, IL_0037, IL_0037, IL_0037)
```

Comment:
It maybe nice to only keep the labels that are actually used, and give them different names (e.g. `lbl_a:`, `lbl_b:`, ...) because of the number is not accurate IL offset anymore after hand-editing.